### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by amarouane-ABDELHAK**
**Summary:** Summary of changes

Addresses deploy blockcahain in production

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -38,35 +38,22 @@ jobs:
...
     with:
       environment: production
       env-file: ".env_prod"
       stage: "production"
       role-session-name: "openscience-blockchain-github-production-deployment"
-      aws-region: "us-west-2"
+      aws-region: "us-east-1"
+
 
     secrets: inherit
```
Comment (xhagrg) [inline .github/workflows/cicd.yml]: New line

**Review by xhagrg — COMMENTED** [review]
Use proper indentations. Left a simple comment. Other things look okay.

File: main.tf [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
-  bc_peer_node_count = max(var.bc_peer_node_count, length(var.ec2_cli_configuration))
-  subnet_id = var.subnet_id
-  vpc_id = var.vpc_id
-  ami_id = var.ami_id
-
-  s3_uri_bc_code = var.s3_uri_bc_code
-
-  docker_file_path          = "./os-blockchain-api/Dockerfile"
-  ecs_container_folder_path = "./os-blockchain-api"
+  source                    = "https://github.com/NASA-IMPACT/tf_aws_managed_blockchain/releases/download/v1.1.6/tf_aws_managed_blockchain.zip"
```
Comment (xhagrg) [inline main.tf]: Still using this version?

File: varaibles.tf [hunk 3]
```diff
@@ -1,47 +1,52 @@
...
+    channel_id       = string
     channel_codename = string
-    instance_type = string
+    instance_type    = string
 
   }))
   default = [
     {
-      channel_id="osbcdevchannel"
+      channel_id       = "osbcdevchannel"
```
Comment (xhagrg) [inline varaibles.tf:11]: osbcdevchannel?

File: varaibles.tf [hunk 4]
```diff
@@ -1,47 +1,52 @@
...
-variable "storage_bucket" {
-
-}
\ No newline at end of file
+variable "peernode_instance_type" {
+  description = "The type of compute instance to use for your peer nodes."
+  default     = "bc.t3.small"
+}
+variable "ecs_domain_name" {
+  default = "osblockchain.net"
```
Comment (xhagrg) [inline varaibles.tf]: move to env variable later.

**Review by xhagrg — APPROVED** [review]
Left some simple comments.

File: main.tf (hunk 2 above)
Comment (amarouane-ABDELHAK) [reply main.tf]: We are using 1.1.9 (flipped the 6)

**Pull request merged**